src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description){
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon(){
```

Comment:
For the subclasses for Task, there are minor space formatting issues for curly braces. If they come after parentheses "()", there should be a space between it and the opening curly brace.

[Reference for this format](https://se-education.org/guides/conventions/java/basic.html#layout)!

Reply:
Thanks for your keen observation, a hindsight on my part!